src/Nemo.jl

```diff
@@ -216,10 +214,15 @@ end
 #
 ################################################################################
 
+const __isthreaded = Ref(false)
+
 function __init__()
    # In case libgmp picks up the wrong libgmp later on, we "unset" the jl_*
    # functions from the julia :libgmp.
-   if __isthreaded
+
+   __isthreaded[] =  "NEMO_THREADED" in keys(ENV) && ENV["NEMO_THREADED"] == "1"
```

Comment:
While you are at it, you migth write it this way, which seems to be slighly faster in case the key is present:
```suggestion
   __isthreaded[] = get(ENV, "NEMO_THREADED", "") == "1"
```